custom-connectors/allGeo/allGeo.swagger.json

```diff
@@ -0,0 +1,543 @@
+{
```

Comment:
@dino-abaqus Can you please let us know the reason for naming the file name to `"allGeo.swagger.json"` instead of keeping it to `"apiDefinition.swagger.json"`? We can not generate connector files with random file names, please keep the file name to `"apiDefinition.swagger.json"`.

Reply:
Updated file name as you requested.